lib/test_process.dart

```diff
@@ -30,13 +30,13 @@ class TestProcess {
   ///
   /// A copy of the underlying stream can be retreived using [stdoutStream].
   StreamQueue<String> get stdout => _stdout;
-  StreamQueue<String> _stdout;
+  late StreamQueue<String> _stdout;
```

Comment:
I think we can use `late final` here to simplify.

```dart
  late final StreamQueue<String> stdout = StreamQueue(stdoutStream());
```

This should let us drop the private `_stdout` and remove line 127. The same can happen with `stderr`.

Reply:
Yep corrected! (for both stderr & stdout), I also fixed some typos